.lando.yml

```diff
@@ -1,9 +1,10 @@
 name: library-main
 recipe: drupal7
 config:
-  php: '7.4'
+  php: '8.0'
   webroot: .
   database: mariadb
+  drush: '8.4.11'
```

Comment:
the default version of drush doesnt work with lando. had to use specific version of drush.